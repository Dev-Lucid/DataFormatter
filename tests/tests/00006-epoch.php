<?php
global $output_path;
$out  = '';
$out .= dfm::construct()->epoch('Y-m-d H:i:s')->apply(1365540913);

file_put_contents($output_path,$out);

?>