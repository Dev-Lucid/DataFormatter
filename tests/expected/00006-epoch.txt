2013-04-09 16:55:13